src/config/lintstagedrc.js

```diff
@@ -1,13 +1,17 @@
 const {resolveHoverScripts, resolveBin} = require('../utils')
 
-const kcdScripts = resolveHoverScripts()
+const hoverScripts = resolveHoverScripts()
 const doctoc = resolveBin('doctoc')
 
 module.exports = {
   'README.md': [`${doctoc} --maxlevel 3 --notitle`],
-  '*.+(js|jsx|json|yml|yaml|css|less|scss|ts|tsx|md|graphql|mdx|vue)': [
-    `${kcdScripts} format`,
-    `${kcdScripts} lint`,
-    `${kcdScripts} test --findRelatedTests`,
+  '*.+(json|yml|yaml|css|less|scss|md|graphql|mdx|vue)': [
+    `${hoverScripts} format`,
+    `${hoverScripts} test --findRelatedTests`,
+  ],
+  '*.+(js|jsx|ts|tsx)': [
+    `${hoverScripts} format`,
+    `${hoverScripts} lint`,
+    `${hoverScripts} test --findRelatedTests`,
```

Comment:
```suggestion
  '*.+(js|jsx|json|yml|yaml|css|less|scss|ts|tsx|md|graphql|mdx|vue)': [
    `${hoverScripts} format`,
  ],
  '*.+(js|jsx|ts|tsx)': [
    `${hoverScripts} lint`,
    `${hoverScripts} test --findRelatedTests`,
```

I think we can split it like this to prevent format/test running twice?